internal/cluster/clustersetup/activity_install_ingress_controller.go

```diff
@@ -1,4 +1,4 @@
-// Copyright © 2018 Banzai Cloud
+// Copyright © 2019 Banzai Cloud
```

Comment:
Oh I see there are divergences already, because NPLS is also called `install` while the cluster autoscaler is called `deploy`, so this kind of invalidates my comment for now (consolidating these is going to be a bigger refactor because the divergence is not being introduced here which I originally assumed).

You can skip this thread.

Reply:
I will rename it to deploy, since autoscaler is newer and you too preferred that.

Reply:
I think it is debatable, because in Helm terminology we are installing deployments and releases and not deploying something, but I'm fine with either names for now as there is going to be at least one outlier.